PVM_PVP/Any/ModernSlayer.py

```
@@ -0,0 +1,493 @@
+def is_spellbook(name):
+    """Check if an item is a spellbook."""
+    name = name.lower()
+    return any(term in name for term in ["spellbook", "scrapper's compendium", "juo'nar's grimoire"])
+
+def slayer_type_from_props(props):
+    """Extract slayer type from item properties."""
+    props = props.lower()
+    if props == "silver":
+        props = "undead slayer"
+    for s in SLAYER_ORDER:
+        sname = re.sub(r'(?<!^)(?=[A-Z])', ' ', [k for k, v in SlayerType.__dict__.items() if v == s][0]).lower()
```

Ruff (0.13.1)

Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)